net-analyzer/ospd-openvas/files/ospd-openvas-add-delay-in-mqtt-test.patch

```diff
@@ -0,0 +1,22 @@
+# Adds a delay to wait for daemon to be started before connect. Gentoo bug 934153
+# https://github.com/greenbone/ospd-openvas/pull/1026
+
+--- a/tests/messaging/test_mqtt.py
++++ b/tests/messaging/test_mqtt.py
+@@ -3,6 +3,7 @@
+ #
+ # SPDX-License-Identifier: AGPL-3.0-or-later
+ 
++import time
+ from datetime import datetime
+ from uuid import UUID
+ 
+@@ -93,6 +94,8 @@ class MQTTDaemonTestCase(TestCase):
+ 
+         daemon.run()
+ 
++        time.sleep(3)
```

Comment:
A delay is fragile, it could still happen on a busy system. Better wait for the TCP port to become accessible. See for example https://github.com/gentoo/gentoo/blob/ca68a18d07f91fcbbe2f57d762ef197c660f7be6/app-shells/atuin/atuin-18.4.0.ebuild#L588-L591